source/geh_calculated_measurements/src/geh_calculated_measurements/net_consumption_group_6/application/calculation.py

```diff
@@ -19,18 +19,20 @@
 @use_span()
 def execute_application(spark: SparkSession, args: NetConsumptionGroup6Args) -> None:
     # Create repositories to obtain data frames
-    electricity_market_repository = ElectricityMarketRepository(spark, args.electricity_market_data_path)
+    electricity_market_repository = ElectricityMarketRepository(spark=spark, catalog_name=args.catalog_name)
```

Comment:
nit: This seems to better align with our common practice
```suggestion
    electricity_market_repository = ElectricityMarketRepository(spark, args.catalog_name)
```